include/sys/zfs_znode.h

```diff
@@ -216,6 +216,7 @@ typedef struct znode {
 	boolean_t	z_is_mapped;	/* are we mmap'ed */
 	boolean_t	z_is_ctldir;	/* are we .zfs entry */
 	boolean_t	z_is_stale;	/* are we stale due to rollback? */
+	uint64_t	z_crtime[2];	/* creation/birth time (cached) */
```

Comment:
Should this also be conditional on `STATX_BTIME`?  If there is no consumer does it make sense to even cache it.

Reply:
I was on the fence on that, but you are right. I will add an #ifdef.

Reply:
As a minor correction to what I said, this was only being set in an older version of the patch. The current version literally doesn't set it at all, such that it really makes no sense. Anyway, I have pushed a revised patch that adds the `#ifdef`.

Reply:
FWIW, it's not like this is going away. stax is a new system call, so Linux is going to have it forever. That said, I don't think the #ifdef is particularly harmful.